test/support/time_helpers.ex

```diff
@@ -0,0 +1,25 @@
+defmodule Castle.TimeHelpers do
+  defmacro __using__(_opts) do
```

Comment:
cooollll

Reply:
Underscores are cool.  The more, the cooler.

Reply:
that is my understanding of underscores as well.